.github/workflows/pull-request.yml

```diff
@@ -41,6 +41,64 @@ jobs:
           command: clippy
           args: --all-targets -- -D warnings
 
+  lint-release:
+    name: Clippy on release profile
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          profile: minimal
+          components: clippy
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: cargo-${{ runner.os }}-${{ github.job }}-${{ hashFiles('**/Cargo.toml') }}
+          restore-keys: |
+            cargo-${{ runner.os }}-${{ github.job }}-
+
+      - name: Run clippy
+        if: always()
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all-targets --release -- -D warnings
+
+  check-release:
```

Comment:
Clippy already runs cargo check, do we really need a separate job for it?

Reply:
Removed.